src/Microsoft.AspNet.Mvc.Razor/Razor/PreCompileViews/PrecompiledViewInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Mvc.Razor
+{
+    /// <summary>
+    /// Represents the result of <see cref="IPrecompiledViewsProvider"/>.
+    /// </summary>
+    public class PrecompiledViewInfo
```

Comment:
Since we now attempt to read resources as assemblies given a `RazorFileInfoCollection`, our previous approach of passing the `IEnumerable<RazorFIleInfoCollection>` to `CompilerCache` ctor was being problematic. The options were having a virtual method in `CompilerCache` that we could override in the test `GetCacheEntriesFromCollections()` or refactoring the view discovery out of the `CompilerCache`. This seemed like a cleaner choice.